components/Hero/src/index.scss

```diff
@@ -147,6 +147,95 @@
   padding-block-end: var(--denhaag-hero-theme-page-title-padding-block-end);
 }
 
+/* added .denhaag-hero to --them-image, otherwise need to use !important to overwrite */
+.denhaag-hero.denhaag-hero--theme-image {
+  background-color: var(--denhaag-hero-theme-image-background-color);
+  background-image: var(--denhaag-hero-theme-image-background-image);
```

Comment:
Er wordt geen background-image gezet in deze modifier. Je hoeft hier geen token voor te maken, dit mag gewoon op `none/unset`.